packages/api/helper/mailer.js

```diff
@@ -96,8 +93,9 @@ function verifyHcEmail(nomination) {
     .send({
       template: 'verifyHcEmail',
       message: {
-        from: formmasterEmail,
-        to: recEmail(nomination.providerEmailAddress),
+        from: nomsEmail,
+        replyTo: nomsEmail,
```

Comment:
I think we can get rid of all of the `replyTo`s. That only matters when the from and reply to fields are different.